fixtures/dom/src/components/fixtures/shadow-dom/index.js

```diff
@@ -0,0 +1,67 @@
+import FixtureSet from '../../FixtureSet';
+import TestCase from '../../TestCase';
+
+const SUPPORTS_SHADOW_DOM = 'attachShadow' in HTMLElement.prototype;
+
+const React = window.React;
+const ReactDOM = window.ReactDOM;
+
+class SelectFixture extends React.Component {
+  render() {
+    if (!SUPPORTS_SHADOW_DOM) {
+      return (
+        <div>Browser does not support Shadow DOM, no tests to execute.</div>
+      );
+    }
+
+    return (
+      <FixtureSet title="Shadow DOM" description="">
+        <TestCase title="Event listeners in shadow-dom" relatedIssues="4963">
+          <TestCase.Steps>
+            <li>Click on the orange box</li>
+          </TestCase.Steps>
+          <TestCase.ExpectedResult>
+            The box should turn green
+          </TestCase.ExpectedResult>
+          <Shadow>
+            <Box />
+          </Shadow>
+        </TestCase>
+      </FixtureSet>
```

Comment:
we should probably also include a `closed` version that ensures documents/asserts the behavior in that case as well